style.css

```diff
@@ -95,6 +95,7 @@ form {
   margin: 0.3em;
 }
 
+.sort-button,
```

Comment:
I love how you just added on another button onto pre-existing css styling for ease and consistency.